package.json

```diff
@@ -345,6 +337,11 @@
             "disabled"
           ],
           "default": "enabled"
+        },
+        "codacy.cli.cliVersion": {
+          "type": "string",
+          "markdownDescription": "The version of the Codacy CLI to use. Check available versions [here](https://github.com/codacy/codacy-cli-v2/releases)",
+          "default": "1.0.0-main.251.1952e73"
```

Comment:
default should be null or undefined (not fixed)

Reply:
Yeah I think at the time Neda defined this with this version because the latest one had issues. I guess we can remove this now